ztools/x509/json.go

```diff
@@ -177,13 +180,18 @@ type jsonSignature struct {
 	SelfSigned         bool               `json:"self_signed"`
 }
 
+type fullValidity struct {
+	validity
+	ValidityPeriod int
```

Comment:
validity is part of tbsCertificate and we'd have to change a bunch of code in x509.go